omod-2.0/src/test/java/org/openmrs/module/unrelatedtest/rest/resource/UnrelatedGenericChildResource.java

```diff
@@ -9,49 +9,50 @@
  */
 package org.openmrs.module.unrelatedtest.rest.resource;
 
-import io.swagger.models.Model;
 import org.openmrs.module.unrelatedtest.UnrelatedGenericChild;
 import org.openmrs.module.webservices.rest.doc.SwaggerSpecificationCreatorTest;
 import org.openmrs.module.webservices.rest.web.RestConstants;
 import org.openmrs.module.webservices.rest.web.annotation.Resource;
 import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
 import org.openmrs.module.webservices.rest.web.v1_0.test.GenericChildResource;
 
 /**
  * A test resource that is unrelated to the main webservices package.
- * 
+ *
  * @see SwaggerSpecificationCreatorTest#testUnrelatedResourceDefinitions()
  */
 @Resource(name = RestConstants.VERSION_1 + "/unrelated", supportedClass = UnrelatedGenericChild.class, supportedOpenmrsVersions = { "1.9.* - 9.*" })
 public class UnrelatedGenericChildResource extends GenericChildResource {
-	
+
 	public static boolean getGETCalled = false;
-	
+
 	public static boolean getCREATECalled = false;
-	
+
 	public static boolean getUPDATECalled = false;
-	
+
 	/*******************************
 	 * TEST METHOD IMPLEMENTATIONS * These methods are the ones we want to test against. There
 	 * implementaion is unimportant, they just set flags so we can assert the methods were called
 	 * correctly by the reflector.
 	 */
-	
+
 	@Override
-	public Model getGETModel(Representation rep) {
+	public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
 		getGETCalled = true;
```

Comment:
Did you intentionally leave in the `getGETCalled`?

Reply:
yes, these may stay mainly because of this test -> https://github.com/mherman22/openmrs-module-webservices.rest/blob/08b817fd98e0c15f81f021d4e856b0c2b598c125/omod-2.0/src/test/java/org/openmrs/module/webservices/rest/doc/SwaggerSpecificationCreatorTest.java#L290-L315